### ISSUE TITLE
Should we keep course on the backend?

### ISSUE THREAD

**Issue by tech4GT** [nsITians/rate.nsut.me#3]
@lakshay365 @gabru-md Rigth now I have not kept course as a table on the database, do we want to store courses as well? I mean shouldn't we allow any course code to be entered, keeping it as string input and not select input?

**Comment by tech4GT** [comment]
It will save a lot effort of adding all courses from all branches!

**Comment by gabru-md** [comment]
@tech4GT you can do it in this way. we can certainly ask the users to add the course, and then we can just save them in lowercase to make sure that proper clustering according to the course is being made , just in case we need it in future for some work or recommendation, maybe ?

**Comment by tech4GT** [comment]
Yeah, I discussed this with @lakshay365 and we agreed on keeping the course on backend as well, we'll keep foreign keys of both course and teacher inside the post! Your thoughts?
